### ISSUE TITLE
Compilation errors with Intel Fortran compiler

### ISSUE THREAD

**Issue by hylandg** [TEOS-10/GSW-Fortran#12]
From a GSW user ...
```
Recently, I've downloaded and am trying to use TEOS10(v3.05, Fortran instead of v3.03) in
my ocean model. However, I met an error when I tried to compile it using Intel fortran (windows).
I found that Intel Fortran allows maximum 511 continuation, but <gsw_mod_saar_data.f90>
has too many continuation line, and it causes compiling error.

I would like to know there is any other method to solve this problem.

Many thanks in advance for your kind answer.
```
A quick online search reveals that this is a well-known limitation with the Intel Fortran compiler (ifort) - the maximum number of continuation lines is hard-coded at 511. Still, much better than the Fortran spec which limits it to 39.

Should be straightforward to fix - will have to modify the script (`make_saar_data.py`) to create data statements that fill multiple short array sections rather than the entire array all at once.

**Comment by hylandg** [comment]
A more detailed online search reveals that this problem has previously been found and apparently fixed by this crowd ... https://github.com/jamesorr/mocsy/issues/6 - who have included a number of GSW routines into their project. :thumbsup:

It's a pity they weren't community-minded enough to let us know about the problem. Although the solution they've adopted is a bit of a hack, and results in an unwieldy `gsw_mod_saar_data.f90` file.

**Comment by hylandg** [comment]
Fixed with commit ec4e9e737bcc3b059dee6f0a3ac184857e3e330f.

Although I don't have access to ifort so I can't absolutely confirm.
